tests/integration/test_dss.py

```diff
@@ -130,6 +130,37 @@ def test_log_command(cleanup_after_initialize) -> None:
     assert "Starting gunicorn" in result.stderr
 
 
+def test_stop_notebook(cleanup_after_initialize) -> None:
+    """
+    Tests that `dss stop` successfully stops a notebook as expected.
+
+    Must be run after `dss create`.
+    """
+    kubeconfig_file = "~/.kube/config"
```

Comment:
I think at this point we can take out this and create a global variable. I see other test cases are using it.